doc/docker-usage.md

```diff
@@ -0,0 +1,45 @@
+# Build a zcashd node from multi-container Docker build
+
+If no VERSION is exported the Dockerfile will apply the default value `master` and build current master.
+
+```
+#in the root of the zcash source directory:
+export VERSION=v2.0.3
+docker build --build-arg VERSION -t zcash/zcash:$VERSION --memory-swap -1 .
+```
+
+# Create docker volumes and populate the `zcash-params` volume
+```
+docker volume create mainnet-params
```

Comment:
Minor, but wouldn't zcash-params or similar be a better volume name given these are not mainnet specific.

Reply:
I had been in the pattern of naming them because recently the testnet params were different. Otherwise 👍